### PR TITLE
Go upgrade  - 1.22.2

### PR DESCRIPTION
**Pull request by deandreJones**
# Description
Upgrade Go 1.20.6 to 122.2

Following instructions - here:
https://dp3.atlassian.net/wiki/spaces/MT/pages/1931673649/How+to+Upgrade+Go+Version

## Changelog or Releases

Add the changelog or release URL for the tool being updated. Some examples below (delete the ones that do not apply
for clarity).

https://go.dev/doc/devel/release#go1.22.minor

### CODE REVIEW

**Review by cameroncaci — APPROVED** [review]
Great to see the build step working again

File: milmove-app/Dockerfile [hunk 1]
```diff
@@ -32,17 +32,17 @@ RUN set -ex && cd ~ \
...
-  # This is so we can pin to specific Node versions
-  # See https://github.com/nodesource/distributions/issues/33#issuecomment-337767815
-  # See https://deb.nodesource.com/node_18.x/pool/main/n/nodejs/ for list of packages
-  && curl -o nodejs.deb https://deb.nodesource.com/node_18.x/pool/main/n/nodejs/nodejs_18.13.0-deb-1nodesource1_amd64.deb \
-  && dpkg -i ./nodejs.deb \
-  && rm nodejs.deb \
+  && : Add Node 18.20.2 \
+  && curl -sSLO https://deb.nodesource.com/setup_18.x \
+ # && echo "${SETUP_18_X_SHA256SUM} setup_18.x" | sha256sum -c - \
+  && bash setup_18.x \
```
Comment (cameroncaci) [inline milmove-app/Dockerfile:44]: We'll want to put the sha256 check back in place as this is an executed file

**Review by cameroncaci — CHANGES_REQUESTED** [review]

**Comment by josiahzimmerman-caci** [comment]
@deandreJones   I think this is good to go to main. We can update the mymove app to the new image

**Comment by deandreJones** [comment]
> @deandreJones I think this is good to go to main. We can update the mymove app to the new image

just need your stamp of approval

**Review by josiahzimmerman-caci — APPROVED** [review]

**Pull request merged**